iot-hub/Samples/device/IoTHubDeviceSamples.sln

```diff
@@ -15,6 +15,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TwinSample", "TwinSample\Tw
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DeviceClientStreamingSample", "DeviceStreamingSample\DeviceClientStreamingSample.csproj", "{B9A6FBA0-23DE-4197-9C3F-BDC4D58BD0DA}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ImportExportDevices", "ImportExportDevicesSample\ImportExportDevices.csproj", "{57DFE1E9-E86D-4C52-A82D-CEE289F5EBF2}"

```

Comment:
Please add a `Run` command in the build.ps1 script to ensure the sample is tested automatically and won't have regressions.

Reply:
I didn't build or modify that script, and don't have confidence about changing something I don't understand. I'm going to merge this branch, and then I can come back around later. I'll contact the IoT guy about it.